terraform/main_pubsub_trigger_gcf.tf

```diff
@@ -0,0 +1,16 @@
+module "cgf_pubsub" {
+  source = "./modules/gcf_gen2_pubsub_source_repo"
+  name = "name_of_cloud_function"
+  description = <<EOF
+  This function will trigger when a new pubsub message is published.
+EOF
+  source_repo_name = var.source_repo_name
+  project = var.project
+  trigger_resource = google_storage_bucket.landing_bucket.name
+  entry_point = "main_cloud_event"
+  environment = {
+    ORDERS_FOLDER = google_storage_bucket_object.orders_folder.name
+    ORDERS_BQ_TABLE = "${google_bigquery_table.orders_table.project}.${google_bigquery_table.orders_table.dataset_id}.${google_bigquery_table.orders_table.table_id}"
```

Comment:
I might be mistaken, but is orders also still specific for a client or is this a general name?

Reply:
This comment went under my radar. Orders here is meant to be just a placeholder. Since many of our clients handle something like "orders" or "purchases" I thought it was generic enough